### PR TITLE
adding ability to remove framework cruft 

### PR DESCRIPTION
**Pull request by kensipe**
from cluster along with zook…eeper nodes

This seems best in packages since it is frameworks that leave this behind.

### CODE REVIEW

File: shakedown/dcos/package.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from dcoscli.package.main import _get_cosmos_url
 
 import shakedown
-
+from shakedown import *
```
Comment (kensipe) [inline shakedown/dcos/package.py]: I'm not a python dev (yet).  I needed functions in __init__ and this seems to be the way to get that to work.  I'm not sure if the import on line 7 is necessary.

**Comment by kensipe** [comment]
This is ready for review / merge.  I've tested it working.

File: shakedown/dcos/package.py [hunk 2]
```diff
@@ -257,3 +257,32 @@ def remove_package_repo(
...
+    principal,
+    zk_path
+):
+    """Delete framework cruft in mesos after an uninstall such as zk nodes
+
+        :param role: name of the role used for reservations
+        :param principal: name of the principal used for reservations or security
+        :param zk_path: path in zookeeper to remove (Without the leading /)
+    """
+    janitor_cmd = 'docker run mesosphere/janitor:verbose /janitor.py'
```
Comment (rji) [inline shakedown/dcos/package.py]: Can we avoid shelling out to a `docker run` here and instead take the relevant functionality from `janitor.py` and build it into this project?

Comment (sschneid) [inline shakedown/dcos/package.py]: Yep, I'm taking a look at this.

> take the relevant functionality from `janitor.py`

For posterity, `janitor.py` lives here: https://github.com/mesosphere/framework-cleaner

**Pull request closed without merging**